### ISSUE TITLE
Cannot Locate the List Box using Xpath

### ISSUE THREAD

**Issue by kirshru** [2gis/Winium#73]
The Inspect properties for this ListBox element is provided in the attachment.  This List box has "" Name , and Dynamic ID (that changes). So , was trying to locate this element using "X Path".

This Element is present on a "Pane" (localizedControlType is Pane, Framworkd - IE browser). The Pane is painted on a Dialog box (Localized ControlType = 'dialog'). The Dialog box is Win 32 window ( FrameworkId : Win32). 

Basically The Dialog box launches a IE browser on which the Pane, and ListBox Controls are present. 

Need solution using X-path to locate the LisBox control.

**Comment by kirshru** [comment]
could not attach a word document with the screen shot.

**Comment by kirshru** [comment]
authenticalassdialog -> is the Dialog  referred in the problem
userauthenticationpane  -> is the pane referred in the problem
mothermaidencombo  -> is the Listbox  --> That has the Name property - blank, Dynamic "AutomationId ".

Tried the following code... Cannot locate "mothermaidencombo  "


  wait.until(ExpectedConditions.visibilityOfElementLocated(By.name("Authentication Assessment Tool")));
        WebElement authenticalassdialog = driver.findElement(By.name("Authentication Assessment Tool"));             

        WebElement userauthenticationpane =  authenticalassdialog.findElement(By.name("User Authentication Module"));
        
        
    //Mothers Maiden Nam or password
      // ID  header-generated9yMoHszoDu
      // NAME - TERRY  
        
  //      WebElement mothermaidencombo =  userauthenticationpane.findElement(By.xpath("button[contains(@id,'header-gen')]"));
